pkg/config/agent/config.go

```diff
@@ -300,6 +300,13 @@ type MulticlusterConfig struct {
 	// clusters directly. This feature also requires Pod CIDRs to be provided in the Multi-cluster Controller
 	// configuration.
 	EnablePodToPodConnectivity bool `yaml:"enablePodToPodConnectivity,omitempty"`
+	// Antrea Multi-cluster WireGuard tunnel configuration.
+	WireGuard WireGuardConfig `yaml:"wireGuard,omitempty"`
+	// Determines how cross-cluster traffic is encrypted.
+	// It has the following options:
+	// - none (default): Cross-cluster traffic will not be encrypted.
+	// - wireguard:      Enable WireGuard for tunnel traffic encryption.
```

Comment:
The option we already used is `wireGuard`.

Reply:
Done, change to `wireGuard`.